otx/algorithms/action/adapters/mmaction/models/detectors/fast_rcnn.py

```diff
@@ -78,8 +86,8 @@ def add_detector(self):
         without pre-proposals. Therefore, when the model infers to dataset without pre-proposal, person detector
         should be added to action detector.
         """
-        detector = ConfigDict(faster_rcnn)
-        self.detector = build_detector(detector)
+        task_processor = build_task_processor(self.detector_cfg, self.deploy_cfg, "cpu")
+        self.detector = task_processor.init_pytorch_model(faster_rcnn_pretrained)
```

Comment:
> Change building method 2d person detector: mmdeploy requires to build model using its processor so that the monkey patched function works properly. Therefore method to add 2d person detector is changed

There is nothing special in `init_pytorch_model` method. This should be reverted.

Reply:
I reverted it. Thanks!!